hoomd_organics/base/simulation.py

```diff
@@ -711,7 +739,8 @@ def _add_hoomd_writers(self):
             filename=self.gsd_file_name,
             trigger=hoomd.trigger.Periodic(int(self.gsd_write_freq)),
             mode="wb",
-            dynamic=["momentum"],
+            dynamic=["momentum", "property"],
+            filter=self.integrate_group,
```

Comment:
I think this should be `hoomd.filter.All()` For the times where `self._integrate_group != hoomd.filter.All()` its just for the purposes of the integrator, but we still want to be able to log and visualize all the particles.